src/test/java/TodoTaskTest.java

```diff
@@ -0,0 +1,32 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TodoTaskTest {
+    @Test
+    @DisplayName("Encoded string of TodoTask is correct")
+    public void testToEncodedString() {
+        // Test setup
```

Comment:
If you want to create a TodoTask before each task, you can consider using JUnit `@BeforeEach` annotation. Reference: https://junit.org/junit5/docs/5.0.2/api/org/junit/jupiter/api/BeforeEach.html#targetText=Annotation%20Type%20BeforeEach&targetText=%40BeforeEach%20is%20used%20to%20signal,in%20the%20current%20test%20class.

Reply:
Thanks for the suggestion - I didn't do it though as I was trying to meme a few messages through the descriptions of each `Task` instantiated in a test.